XLForm/XL/Cell/XLFormBaseCell.h

```diff
@@ -35,7 +35,7 @@
 
 @property (nonatomic, weak) XLFormRowDescriptor * rowDescriptor;
 
--(XLFormViewController *)formViewController;
+@property (nonatomic, readonly, strong) XLFormViewController *formViewController;
```

Comment:
>I think you can even use dot-syntax with methods without parameters.

Good point. Dot-syntax is just syntax, it has nothing to do with `@property`, so dot-syntax will already work as-is as well as `valueForKey:`. 

> I mean, the method is more clear showing the real propose of it, i get confused when i find a property that is not actually a property (Does the instance has a internal state to save it or not? ).

The only comment I can make on this is whether properties are backed by an ivar is irrelevant to the objects which interact with it.

>BTW strong doesn't apply to readonly properties without accessor method. I want to avoid this confusion and make it clear ;).

I assume you mean `strong` doesn't apply to read-only properties without an ivar, and that is correct. The qualifier `strong` ought to be omitted anyway since it the default option.